src/slash-bedrock/share/brl-fetch/distros/funtoo

```diff
@@ -0,0 +1,85 @@
+#!/bedrock/libexec/busybox sh
+#
+# Funtoo Linux bootstrap support
+#
+#      This program is free software; you can redistribute it and/or
+#      modify it under the terms of the GNU General Public License
+#      version 2 as published by the Free Software Foundation.
+#
+# Copyright (c) 2022-2022 Nicolas Lorin <androw95220@gmail.com>
+#
+
+# shellcheck source=src/slash-bedrock/libexec/brl-fetch
+. /bedrock/share/common-code
+trap 'fetch_abort "Unexpected error occurred."' EXIT
+
+check_supported() {
+        true
+}
+
+speed_test_url() {
+        echo "index.xml"
+}
+
+list_mirrors() {
+        echo "https://build.funtoo.org/"
+}
+
+brl_arch_to_distro() {
+        case "${1}" in
+        "aarch64") echo "arm64_generic" ;;
+        "armv7hl") echo "armv7a_vfpv3_hardfp" ;;
+        "armv6j") echo "armv6j_hardfp" ;;
+        "i686") echo "i686" ;;
+        "x86_64") echo "generic_64" ;;
+        *) abort "brl does not know how to translate arch \"${1}\" to ${distro} format" ;;
+        esac
+}
+
+list_architectures() {
+        cat <<EOF
+aarch64
+armv7hl
+armv6j
+i686
+x86_64
+EOF
+}
+
+default_release() {
+        echo "1.4-release-std"
+}                           
+
+list_releases() {
+        cat <<EOF
+1.4-release-std
+next
```

Comment:
Is it not feasible to dynamically detect the release list?  Will we have to manually update this come 1.5 or 2.0?

Reply:
Done for both list and default!